File: src/main/java/io/github/eone666/telegramjoinnotifications/events/PlayerQuit.java
package io.github.eone666.telegramjoinnotifications.events;

import io.github.eone666.telegramjoinnotifications.utils.Telegram;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerQuitEvent;

public class PlayerQuit implements Listener {

    private Telegram _tg = null;
    public PlayerQuit(Telegram tg){
        _tg = tg;
    }
    @EventHandler
    public void onPlayerQuit(PlayerQuitEvent event) {
        String playerName = event.getPlayer().getName();
        _tg.SendMessage(String.format("%s has left the game",playerName ));
    }
}

File: src/main/kotlin/io/github/eone666/telegramnotifier/listeners/PlayerQuit.kt
package io.github.eone666.telegramnotifier.listeners

import io.github.eone666.telegramnotifier.pluginInstance
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerQuitEvent

class PlayerQuit() : Listener {
    @EventHandler
    suspend fun onPlayerQuit(event: PlayerQuitEvent) {
        if(pluginInstance.config.isPluginConfigured.boolean){
            pluginInstance.notifications.quit(event.player)
            pluginInstance.playersList.remove(event.player)
        }
    }
}